main.c

```diff
@@ -162,6 +162,12 @@ main(int argc, char *argv[])
 		}
 	} else {
 		infile = stdin;
+
+		/* Error if given incompatible options. */
+		if (devtty == 0) {
+			warn0("Cannot combine -P and infile=-");
```

Comment:
This would look less "punctuation-y" if we wrote "infile=stdin", but what the user actually would have written on the command-line is "`scrypt dec -P -`".

Reply:
How about "Cannot read both passphrase and input file from standard input"?